### PR TITLE
Format tx process email templates

### PR DESCRIPTION
**Pull request by lyyder**
Fixes html indentation on tx process email templates.

### CODE REVIEW

**Comment by Gnito** [comment]
Could you add a new entry to [CHANGELOG.md](https://github.com/sharetribe/web-template/blob/main/CHANGELOG.md#upcoming-version-2023-xx-xx)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ way to update this template, but currently, we follow a pattern:
   timing of the complete transition [#142](https://github.com/sharetribe/web-template/pull/142)
 - [fix] Email templates: fix layout-breaking condition in purchase templates.
   [141](https://github.com/sharetribe/web-template/pull/141)
+- [fix] Fix indentation in default transaction process email templates.
+  [#140](https://github.com/sharetribe/web-template/pull/140)
```
Comment (Gnito) [inline CHANGELOG.md:24]: @lyyder We try to follow the merge order here - so, this should have been the first one on the list.
That being said, no need to change it.

**Review by Gnito — APPROVED** [review]
Too many changes to be able to fully review this, but LGTM :/

File: ext/transaction-processes/default-booking/templates/booking-accepted-request/booking-accepted-request-html.html [hunk 2]
```diff
@@ -1,234 +1,232 @@
...
+    <tbody>
+      <tr>
+        <td>
+          <table align="center" role="presentation" cellSpacing="0" cellPadding="0" border="0" width="100%" style="max-width:600px;margin:0 auto">
+            <tr style="width:100%">
+              <td>
+                <h1 style="color:#484848;font-size:26px;line-height:1.3;font-weight:700">Your booking request was accepted</h1>{{#each tx-line-items}}{{#eq "line-item/hour" code}}
+                <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">{{ provider.display-name }} has accepted your booking request for {{ listing.title }} from {{> format-date date=booking.start}} to {{> format-date date=booking.end}}.</p>
+                {{/eq}}{{#eq "line-item/day" code}}
+                <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">{{ provider.display-name }} has accepted your booking request for {{ listing.title }} from {{> format-date- ay date=booking.start}} to {{> format-date-day-before date=booking.end}}.</p>
```
[the remark this answers is not in the document]
Comment (lyyder) [reply ext/transaction-processes/default-booking/templates/booking-accepted-request/booking-accepted-request-html.html:16]: Nice catch!

**Pull request merged**